src/ol/overlay.js

```diff
@@ -357,6 +380,9 @@ goog.exportProperty(
  */
 ol.Overlay.prototype.setPosition = function(position) {
   this.set(ol.OverlayProperty.POSITION, position);
+  if (goog.isDef(position) && this.autoPan_) {
+    this.panIntoView_();
+  }
```

Comment:
Everywhere else, for `ol.Object` properties, we use `change:` listeners instead of adding code to the setter (`setPosition` in this case). Using a `change:` listener `panIntoView_` will still be called if `set('position', value)` is used instead of `setPosition(value)` for example.